src/tv2-common/migrations/index.ts

```diff
@@ -125,6 +129,61 @@ export function AddGraphicToGFXTable(
 	}
 }
 
+export function mapGFXTemplateToDesignTemplate(versionStr: string, studio: string, from: string, to: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.mapGFXTemplateToDesignTemplate.${from}.${studio}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[] | undefined
+
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[] | undefined
+
+			if (!gfxTemplates || !gfxTemplates.length) {
+				return false
+			}
+
+			if (designTemplates && designTemplates.length) {
+				return false
+			}
+
+			return gfxTemplates.some(g => g.IsDesign)
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[]
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[]
```

Comment:
Will the `to` table exist? It probably won't. I got`Error in Migration step blueprint_tv2_afvd_showstyle_showStyle_guKzgK4bOXOy0FrF6_1.7.5.mapGFXTemplateToDesignTemplate.GFXTemplates.AFVD: Cannot read property 'push' of undefined, TypeError: Cannot read property 'push' of undefined`
```suggestion
			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[] ?? []
```

Reply:
This has been implemented now! Although i never got that error somehow :-)